bodhi/server/models.py

```diff
@@ -1259,6 +1272,41 @@ def signed(self):
             return True
         return all([build.signed for build in self.builds])
 
+    @property
+    def ci_status(self):
+        """ Returns a boolean representing if all the builds associated with
+        this update are ignored or have passed their CI testing.
+
+        Returns False if at least one of the tests of the builds associated
+        with this update are queued, running or failed.
+        """
+
+        ci_status = set([build.ci_status for build in self.builds])
+        if not ci_status.difference(set([CiStatus.ignored, CiStatus.passed])):
+            return True
+        return False
```

Comment:
> One more thought - it might be good to allow None to mean "passed" too.

We agreed that then there is no distinction between: ``I do not know whether this update should be tested`` and ``I missed the info about it being ignored``, as well as potentially run into race-condition issue between the time the update is creating and the time it is checked for CI for inclusion into the mash process.